nebullvm/api/functions.py

```diff
@@ -199,6 +199,8 @@ def optimize_model(
     FEEDBACK_COLLECTOR.start_collection(model, framework=dl_framework)
     if metric_drop_ths is not None and metric_drop_ths <= 0:
         metric_drop_ths = None
+    elif metric_drop_ths is not None:
```

Comment:
shouldn't we check if `metric` is None as well?

Reply:
Yep you are right